common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -36,6 +36,11 @@ object Messages {
     deviceId: Option[DeviceId],
     deviceType: DeviceType) extends BusMessage
 
+  final case class DevicePublicCredentialsSet(
+    namespace: Namespace,
```

Comment:
All our events current have a namespace, and many of our services don't know which namespace a device belong to, so I included the namespace.